common/beerocks/bwl/CMakeLists.txt

```diff
@@ -83,13 +98,50 @@ elseif(BWL_TYPE STREQUAL "WAV")
 
 elseif(BWL_TYPE STREQUAL "DWPAL")
     
+    include_directories(
+        ${HOSTAPD_DIR}/src/drivers
+        ${PLATFORM_BUILD_DIR}/iwlwav-iw-4.14
+        ${PLATFORM_INCLUDE_DIR}/libnl3
+    )
+
     file(GLOB bwl_platform_sources
         ${bwl_platform_sources}
         ${MODULE_PATH}/dwpal/*.c*
     )
 
-    find_package(dwpal REQUIRED)
-    list(APPEND BWL_LIBS dwpal::dwpal)
+    # find_package(dwpal REQUIRED)
+    list(APPEND BWL_LIBS dwpal)
+
+elseif(BWL_TYPE STREQUAL "NL80211")
+
+    file(GLOB bwl_platform_sources
+        ${MODULE_PATH}/nl80211/*.c*
+
+        # WPA Control Interface
+        ${HOSTAPD_DIR}/src/common/wpa_ctrl.c
+        ${HOSTAPD_DIR}/src/utils/os_unix.c
+    )
+
+    # Enable WPA control interface
+    add_definitions(
+        -DCONFIG_CTRL_IFACE
+        -DCONFIG_CTRL_IFACE_UNIX
+    )
+
+    # Hostapd/NL80211 include directories
+    include_directories(
+        ${HOSTAPD_DIR}/src/utils
+        ${HOSTAPD_DIR}/src/common
+        ${HOSTAPD_DIR}/src/drivers
+        ${PLATFORM_INCLUDE_DIR}/libnl-tiny
```

Comment:
Why?

Reply:
Because `PLATFORM_INCLUDE_DIR` is a hack that makes it more difficult to integrate prplMesh in build systems and distros. Adding more hacks like this is just making things worse.

OTOH, it is already done like that in bpl, so I'm OK with keeping it like this (and fixing both together).